Translations/pt_BR.po

```diff
@@ -794,443 +794,437 @@ msgid "Read-Only Directory Error"
-msgstr ""
+msgstr "Pergaminho de %s"
 
 #: Source/control.cpp:467 Source/control.cpp:1063
 msgid "1 Scroll"
-msgstr ""
+msgstr "1 pergaminho"
 
 #: Source/control.cpp:469 Source/control.cpp:1065
 #, c-format
 msgid "%i Scrolls"
-msgstr ""
+msgstr "%i pergaminhos"
 
 #: Source/control.cpp:473 Source/control.cpp:1069 Source/items.cpp:1594
 #, c-format
 msgid "Staff of %s"
-msgstr ""
+msgstr "Cajado de %s"
 
 #: Source/control.cpp:475 Source/control.cpp:1072
 msgid "1 Charge"
-msgstr ""
+msgstr "1 Carga"
 
 #: Source/control.cpp:477 Source/control.cpp:1074
 #, c-format
 msgid "%i Charges"
-msgstr ""
+msgstr "%i Cargas"
 
 #: Source/control.cpp:486
 #, c-format
 msgid "Spell Hotkey #F%i"
-msgstr ""
+msgstr "Tecla de atalho #F%i"
 
 #: Source/control.cpp:1005
 msgid "Player friendly"
-msgstr ""
+msgstr "Amigável a jogadores"
 
 #: Source/control.cpp:1007
 msgid "Player attack"
-msgstr ""
+msgstr "Hostil a jogadores"
 
 #: Source/control.cpp:1010
 #, c-format
 msgid "Hotkey: %s"
-msgstr ""
+msgstr "Tecla de atalho:%s"
 
 #: Source/control.cpp:1019
 msgid "Select current spell button"
-msgstr ""
+msgstr "Selecionar feitiço atual"
 
 #: Source/control.cpp:1023
 msgid "Hotkey: 's'"
-msgstr ""
+msgstr "Tecla de atalho:'s'"
 
 #: Source/control.cpp:1261 Source/inv.cpp:2120 Source/items.cpp:3075
 #, c-format
 msgid "%i gold %s"
-msgstr ""
+msgstr "%i %s de ouro"
 
 #: Source/control.cpp:1264
-#, fuzzy
-#| msgid "Requirements:"
 msgid "Requirements not met"
-msgstr "Requisitos:"
+msgstr "Requisitos não atendidos"
 
 #: Source/control.cpp:1300
-#, fuzzy, c-format
-#| msgid "Level:"
+#, c-format
 msgid "%s, Level: %i"
-msgstr "Nível:"
+msgstr "%s, Nível: %i"
 
 #: Source/control.cpp:1302
 #, c-format
 msgid "Hit Points %i of %i"
-msgstr ""
+msgstr "Pontos de vida %i de %i"
 
 #: Source/control.cpp:1375
 msgid "None"
-msgstr ""
+msgstr "Nenhum"
 
 #: Source/control.cpp:1443 Source/control.cpp:1455 Source/control.cpp:1467
 #, c-format
 msgid "MAX"
-msgstr ""
+msgstr "MÁX"
 
 #: Source/control.cpp:1841
 msgid "Skill"
-msgstr ""
+msgstr "Habilidade"
 
 #: Source/control.cpp:1844
 #, c-format
 msgid "Staff (%i charges)"
-msgstr ""
+msgstr "Cajado (%i cargas)"
 
 #: Source/control.cpp:1852
 #, c-format
 msgid "Mana: %i  Dam: %i - %i"
-msgstr ""
+msgstr "Mana:%i  Dano:%i - %i"
 
 #: Source/control.cpp:1854
 #, c-format
 msgid "Mana: %i   Dam: n/a"
-msgstr ""
+msgstr "Mana:%i   Dano:n/d"
 
 #: Source/control.cpp:1857
 #, c-format
 msgid "Mana: %i  Dam: 1/3 tgt hp"
-msgstr ""
+msgstr "Mana:%i  Dano:1/3 vida"
 
 #: Source/control.cpp:1902
 msgid "piece"
-msgstr ""
+msgstr "peça"
 
 #: Source/control.cpp:1904
 msgid "pieces"
-msgstr ""
+msgstr "peças"
 
 #: Source/control.cpp:1912
-#, fuzzy, c-format
-#| msgid "You do not have enough gold"
+#, c-format
 msgid "You have %u gold"
-msgstr "Você não tem ouro o suficiente"
+msgstr "Você tem %u de ouro"
 
 #: Source/control.cpp:1914
 #, c-format
 msgid "%s.  How many do"
-msgstr ""
+msgstr "%s.  Quanto quer"
```

Comment:
"Quantos"

Reply:
it's much easier if you make a proper suggestion, which allows it to get applied from github's website
![image](https://user-images.githubusercontent.com/14297035/116017138-0ad7d700-a63f-11eb-907e-b6e12502d794.png)